wallet-new/src/Cardano/Wallet/Client.hs

```diff
@@ -62,13 +66,19 @@ data WalletClient m
          :: WalletId -> PasswordUpdate -> Resp m Wallet
     , deleteWallet
          :: WalletId -> Resp m ()
+    , getWallets
+         :: Resp m [Wallet]
     , getWallet
          :: WalletId -> Resp m Wallet
     , updateWallet
          :: WalletId -> Update Wallet -> Resp m Wallet
     -- account endpoints
+    , postAccount
+         :: New Account -> Resp m Account
     , deleteAccount
          :: WalletId -> AccountIndex -> Resp m ()
+    , getAccounts
+         :: WalletId -> Resp m [Account]
```

Comment:
`getAccounts` will need to be a top level alias; the actual function will look something like

```haskell
getAccountsPaged
    :: WalletId -> Maybe Page -> Maybe PerPage -> Resp m [Account]
```

Reply:
You can change this now or I can change it when I merge the client implementation in.

Reply:
You can change it, I would prefer if we drop this and use your `streamA :: WalletClient m -> Producer m A`.

Reply:
I can implement that on top of the paginated version, but the paginated version will be in the client record as a base.